src/Prometheus/Storage/Adapter.php

```diff
@@ -37,10 +45,21 @@ public function updateGauge(array $data): void;
     public function updateCounter(array $data): void;
 
     /**
-     * Removes all previously stored metrics from underlying storage
-     *
-     * @throws StorageException
+     * @param mixed[] $data
      * @return void
      */
-    public function wipeStorage(): void;
+    public function initHistogram(array $data): void;
```

Comment:
Adding new methods to this interface is a breaking change, we should avoid doing it or maybe release this as a new version afterward (lesson learned from #43)